addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -134,25 +134,22 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
     }
 
     // Ribbon for promotions on products
-    .ribbon-wrapper {
-        display: none;
-        width: 85px;
-        height: 88px;
-        z-index: 5;
-        overflow: hidden;
-        @include o-position-absolute($left: 0, $top: 0);
-    }
-    .ribbon {
-        font: bold 15px Sans-Serif;
-        color: white;
+    .o_wribbon {
```

Comment:
`o_wribbon`? Either `o_wsale_ribbon` or `o_ribbon` as a generic website style.

Reply:
Do you think we might want it as a generic website style? I can maybe see it used for website snippet options in the future so maybe?

Reply:
In fact, already used by events (example on /event/design-fair-los-angeles-2020-03-28-2020-04-01-1/page/introduction-design-fair-los-angeles). Could be good to share the same style in this PR directly. So yes, generic class in website.scss.

Reply:
In event, the ribbon is not in the same corner. Should I use the bootstrap system with one class for the common style and one for the specific style? (o_ribbon, o_ribbon_top_left, o_ribbon_top_right, ...)

Reply:
Yes. Or move the event ones at the top-left, your call :+1:

Reply:
Actually it seems like we have ribbon implementations all over the place. Should I refactor all the frontend ones to use this new implementation and leave the backend ones (eg, Archived, Paid invoices) untouched? (those use a widget to change the font size to make it fit inside the ribbon)

Reply:
Yes. If you want, you can ping the framework team once the task is merged to tell them we have better ribbons if they want to get inspired :)

Reply:
In the end, there was only actual ribbons in event, so I just overrode the transform properties there so it's in the top right (top left was hiding too much of the sponsor logos). There are a few things though that have ribbon in their name but none of the ribbon mechanics, or are in the backend, so I didn't touch those.